From bca66685ebc6cd236a8abecb1942d218113638c4 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Sun, 29 Mar 2009 11:03:25 +0200
Subject: [PATCH] arch/m68k/emu/nfeth.c doesn't need obsolete <net/ieee80211.h>

commit f3734ee6df3ac57151e02d091f47d5e52e646539 ("make net/ieee80211.h private
to ipw2x00") removed <net/ieee80211.h>, breaking the build of
arch/m68k/emu/nfeth.c. Remove the #include, as it's not needed anyway.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 arch/m68k/emu/nfeth.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/m68k/emu/nfeth.c b/arch/m68k/emu/nfeth.c
index de7cdad..5b118e9 100644
--- a/arch/m68k/emu/nfeth.c
+++ b/arch/m68k/emu/nfeth.c
@@ -12,7 +12,6 @@
 #include <linux/netdevice.h>
 #include <linux/etherdevice.h>
 #include <linux/module.h>
-#include <net/ieee80211.h>
 #include <asm/natfeat.h>
 #include <asm/virtconvert.h>
 
-- 
1.5.6.5

